packages/edit-site/src/components/sidebar-navigation-screen-main/index.js

```diff
@@ -76,10 +95,20 @@ export default function SidebarNavigationScreenMain() {
 		<SidebarNavigationScreen
 			isRoot
 			title={ __( 'Design' ) }
-			description={ __(
-				'Customize the appearance of your website using the block editor.'
-			) }
-			content={ <MainSidebarNavigationContent /> }
+			description={
+				isBlockBasedTheme
+					? __(
+							'Customize the appearance of your website using the block editor.'
+					  )
+					: __(
+							'Explore block styles and patterns to refine your site'
+					  )
+			}
+			content={
+				<MainSidebarNavigationContent
+					isBlockBasedTheme={ isBlockBasedTheme }
```

Comment:
Maybe you saw this already, but I think this prop is missing from here as well:

https://github.com/WordPress/gutenberg/blob/bc91ba9d0cbc4b63b1fc02df7f99481d47f02246/packages/edit-site/src/components/sidebar-navigation-screen-global-styles/index.js#L80-L80

Without it, block themes will use the classic route:


https://github.com/user-attachments/assets/d5f93d46-63a5-4670-899c-4f33336c8d46


Also, to prevent the nav animating, the correct route needs to be used in layout:

https://github.com/WordPress/gutenberg/blob/bc91ba9d0cbc4b63b1fc02df7f99481d47f02246/packages/edit-site/src/components/layout/index.js#L135-L135

So I think it's now `styles` in this PR?

Reply:
> I think this prop is missing from here as well

Fixed!

> So I think it's now styles in this PR?

That route changed in #67199; in this PR I only introduced the new `stylebook` route. 

What is actually supposed to animate (or not animate) in this case? I'm testing on trunk but can't see any difference when changing the routeKey.

Reply:
> What is actually supposed to animate (or not animate) in this case? I'm testing on trunk but can't see any difference when changing the routeKey.

Oh sorry, there's a regression caused by https://github.com/WordPress/gutenberg/pull/67057

My bad. 

The panel isn't supposed to animate if the items don't change.

I'll fix that in a new PR.